src/scss/style.scss

```diff
@@ -0,0 +1,5 @@
+// Import styles
+@import "../../node_modules/@coreui/coreui/scss/coreui";
+
+// Temp fix for reactstrap
```

Comment:
create a story in the backlog to remove the temp fix explaining when it can be removed + add link to the story in this comment + prefix the comment with `TODO: `

Reply:
The temp fix was added by CoreUI devs to fix a problem with right dropdown when used in app navbar
The issue can be reproduced in https://github.com/coreui/coreui-free-react-admin-template, when the fix is commented out, the dropdown menu that is shown from user avatar has the wrong width (.app-header .dropdown-item has min width of 180px and overflows menu)
Should be fixed by CoreUI devs or will be fixed with further analysis later
Opened story https://www.pivotaltracker.com/story/show/167402768 to keep track and added TODO prefix to comment

Reply:
add the link to the story in the comment below the todo